src/modules/dataFetching/components/DeleteMutationButton.tsx

```diff
@@ -108,32 +61,11 @@ const DeleteMutationButton = <Mutation, MutationVariables>({
           {children}
         </Button>
       )}
-      <ConfirmationDialog
-        id='deleteRecordDialog'
+      <DeleteMutationConfirmationDialog
         open={showDialog}
-        title={`${capitalize(verb)} ${recordName}`}
-        confirmText={`Yes, ${verb} ${recordName}`}
-        cancelText='Close'
-        onConfirm={handleDelete}
-        onCancel={() => setShowDialog(false)}
-        loading={loading}
-        errorState={errorState}
-        color='error'
-        {...ConfirmationDialogProps}
-      >
-        <>
-          {confirmationDialogContent ? (
-            confirmationDialogContent
-          ) : (
-            <>
-              <Typography>
-                {`Are you sure you want to ${verb} this ${recordName}?`}
-              </Typography>
-              <Typography>This action cannot be undone.</Typography>
-            </>
-          )}
-        </>
-      </ConfirmationDialog>
+        onClose={() => setShowDialog(false)}
+        {...rest}
```

Comment:
should pass through `recordName` as well, otherwise deletion confirmation models just show "
are you sure you want to delete this record"

Reply:
Thanks, good catch!